docs/en/get-started/index.rst

```diff
@@ -620,5 +620,10 @@ Related Documents
     toolchain-setup-scratch
     :esp32: ../get-started-legacy/index
 
+Using with PlatformIO
+=====================
+
+.. include:: using-platformio.rst
+    
```

Comment:
Seems to be left over from the previous version?

Reply:
When fixing this, would you mind squashing the commits into one? (If not, it's also fine — can squash them when merging this internally)

Reply:
Thanks, fixed!